src/main/resources/db/cassandra/001_Initialize_db.cql

```diff
@@ -23,24 +23,32 @@ CREATE TABLE IF NOT EXISTS repair_unit (
   column_families set<text>,
   incremental_repair boolean
 )
-  WITH compaction = {'class': 'LeveledCompactionStrategy'}
-  AND caching = {'rows_per_partition': 10};
+  WITH compaction = {'class': 'LeveledCompactionStrategy'};
 
 CREATE TABLE IF NOT EXISTS repair_run (
-  id                 timeuuid PRIMARY KEY,
-  cluster_name       text,
-  repair_unit_id     timeuuid,
-  cause              text,
-  owner              text,
-  state              text,
-  creation_time      timestamp,
-  start_time         timestamp,
-  end_time           timestamp,
-  pause_time         timestamp,
-  intensity          double                    ,
-  last_event         text                    ,
-  segment_count      int                     ,
-  repair_parallelism text
+  id                 timeuuid,
+  cluster_name       text      STATIC,
+  repair_unit_id     timeuuid    STATIC,
+  cause              text      STATIC,
+  owner              text      STATIC,
+  state              text      STATIC,
+  creation_time      timestamp STATIC,
+  start_time         timestamp STATIC,
+  end_time           timestamp STATIC,
+  pause_time         timestamp STATIC,
+  intensity          double    STATIC,
+  last_event         text      STATIC,
+  segment_count      int       STATIC,
+  repair_parallelism text      STATIC,
+  segment_id         timeuuid,
+  start_token        varint,
+  end_token          varint,
+  segment_state      int,
+  coordinator_host   text,
+  segment_start_time timestamp,
+  segment_end_time   timestamp,
+  fail_count         int,
+  PRIMARY KEY (id, segment_id)
 )
   WITH compaction = {'class': 'LeveledCompactionStrategy'}
   AND caching = {'rows_per_partition': 10};
```

Comment:
I still think we need to make no change that requires the users to drop the database then recreate it.
As schema migration is performed, we need to create a new cql file with a new version number, create a single table in it that we would call `repair_run_v2` for example, and move all queries from `repair_run` to `repair_run_v2` in the code. Otherwise the app will just crash if people with existing installs try to upgrade.

We also need more caching here since many runs could have thousands of segments. After running some tests on the old schema, setting 1000 rows_per_partition uses only a few kb in cache so I guess we can use values ranging between 1000 and 5000 to maximize cache utilization.